queries.py

```diff
@@ -93,7 +93,7 @@
         ATD_SUBPROJECT_FDU_VW
```

Comment:
Out of pure curiosity, do you know what other dependencies this other FDU view and [its dataset](https://data.austintexas.gov/Transportation-and-Mobility/Transportation-Capital-Project-Financial-Codes/jega-nqf6/data_preview) have besides Moped? 

I guess that I'm trying to think ahead about taking advantage of the new availability of the program tagging in this new dataset and also about how different these two datasets really are.

Reply:
hmmm I think this [data](https://data.austintexas.gov/Transportation-and-Mobility/Transportation-Capital-Project-Financial-Codes/jega-nqf6/about_data) appears to be by subproject, rather than FDU? It doesn't look like the [DAG](https://github.com/cityofaustin/atd-airflow/blob/production/dags/atd_finance_data_fdus.py) is sending data to Knack so I believe Moped is the only place this data is going?